src/internal_modules/roc_core/pool.h

```diff
@@ -23,6 +23,11 @@
 namespace roc {
 namespace core {
 
+//! Memory pool flags.
+enum PoolFlags {
+    PoolFlags_DisableOverflowPanic = 0x01, //! Disables overflow panic and logs instead.
+};
```

Comment:
A few nitpicks:

* Let's rename it from `PoolFlags_DisableOverflowPanic` to `PoolFlag_DisableOverflowPanic`, because it's a name for single flag.

* Usually we use `(1 << N)` instead of `0xM` to define flags.

* Negations are a bit harder to handle when reading code. So I'd suggest to add something like `PoolFlag_PanicOnOverflow` or `PoolFlags_OverflowPanics` and use it in default ctor argument.

* If doxygen comment is after declaration, IIRC it should be `//!<` instead of `//!`.

Reply:
Sorry. Still adapting to the project.